src/components/Drum.js

```diff
@@ -0,0 +1,41 @@
+import DrumPad from "./DrumPad";
+import React, { useEffect } from 'react';
+
+function Drum(props) {
+  const { kit } = props;
+
+  useEffect(() => {
+    document.addEventListener('keydown', handleKeyDown);
```

Comment:
Not a good practice to access the DOM addEventListener and removeEventListener, react has their own way of handling events